cmd/agent/daemon/daemon.go

```diff
@@ -71,11 +71,14 @@ func NewRunCommand(version string) *cobra.Command {
 			},
 		}
 		ebpfEventsStdioExporterEnabled = command.Flags().Bool("ebpf-events-stdio-exporter-enabled", false, "Export ebpf event to stdio")
-		ebpfEventsPerCPUBuffer         = command.Flags().Int("ebpf-events-per-cpu-buffer", os.Getpagesize()*64, "Ebpf per cpu buffer size")
 		ebpfEventsOutputChanSize       = command.Flags().Int("ebpf-events-output-queue-size", 4096, "Ebpf user spaces output channel size")
-		ebpfTracerMetricsEnabled       = command.Flags().Bool("ebpf-tracer-metrics-enabled", false, "Enables the export of tracer related metrics from eBPF")
+		ebpfTracerMetricsEnabled       = command.Flags().Bool("ebpf-tracer-metrics-enabled", true, "Enables the export of tracer related metrics from eBPF")
 		ebpfProgramMetricsEnabled      = command.Flags().Bool("ebpf-program-metrics-enabled", false, "Enables the export of metrics about eBPF programs")
 
+		EBPFSignalEventsRingBufferSize = command.Flags().Uint32("ebpf-signal-events-ring-buffer-size", 1<<20, "Ebpf ring buffer size in bytes for priority events")
```

Comment:
We need some sort of validation, as the value needs to be a power of `2`.

Reply:
But it tells a proper error already during the crash if it's not power of 2.

Reply:
There are many flags which would fail if the agent if passed with incorrect values. Not sure if this is a good scope to add custom validations to flags.